markdown/ch4.markdown

```diff
@@ -1075,22 +1148,26 @@ positive or a negative frequency. This component of the DFT represents
 the Nyquist Frequency, i.e. half of the sampling frequency, and is
 useful to orientate oneself when looking at DFT graphics.
 
+[^odd_N]: We leave it as an exercise for the reader to picture the
+          situation for $N$ odd.  In this chapter, all examples use
+          even-order DFTs.
+
 The FFT in turn is simply a special and highly efficient algorithm for
 calculating the DFT. Whereas a straightforward calculation of the DFT
 takes of the order of $N^{2}$ calculations to compute, the FFT
 algorithm requires of the order $N\log N$ calculations. The FFT was
 the key to the wide-spread use of the DFT in real-time applications
-and was included in a list of the top 10 algorithms of the $20^{th}$
+and was included in a list of the top $10$ algorithms of the $20^{th}$
 century by the IEEE journal Computing in Science & Engineering in the
-year 2000.
+year $2000$.
```

Comment:
I don't think math font is required for the year or for inline numbers not referring to the mathematics.